openforcefield/utils/toolkits.py

```diff
@@ -1523,11 +1526,24 @@ class RDKitToolkitWrapper(ToolkitWrapper):
 
     .. warning :: This API is experimental and subject to change.
     """
+
     _toolkit_name = 'The RDKit'
     _toolkit_installation_instructions = 'A conda-installable version of the free and open source RDKit cheminformatics ' \
                                          'toolkit can be found at: https://anaconda.org/rdkit/rdkit'
     _toolkit_file_read_formats = ['SDF', 'MOL', 'SMI']  #TODO: Add TDT support
-    _toolkit_file_write_formats = ['SDF', 'MOL', 'SMI', 'PDB']
+
+    # Note any new file write formats should be added here with a string of class/method used
+    _toolkit_file_write_formats = {'SDF': 'Chem.SDWriter', 'MOL': 'Chem.SDWriter', 'SMI': 'Chem.SmilesWriter',
```

Comment:
This list now only has string representations of the functions which can later be called using eval, so when someone updates the writers to add more functionality in future they only have to update this dictionary for RDKit.